ts/modules/draggable-points.src.ts

```diff
@@ -2350,15 +2352,15 @@ H.Point.prototype.getDropValues = function (
     const limitToRange = function (val: number, direction: string): number {
         var defaultPrecision = (series as any)[direction.toLowerCase() + 'Axis']
                 .categories ? 1 : 0,
-            precision = pick<number>(
+            precision = pick(
                 (options as any)['dragPrecision' + direction], defaultPrecision
             ),
-            min = pick<number>(
-                (options as any)['dragMin' + direction],
+            min = pick<number, number>(
+                (options as any)['dragMin' + direction] as any,
                 -Infinity
             ),
-            max = pick<number>(
-                (options as any)['dragMax' + direction],
+            max = pick<number, number>(
+                (options as any)['dragMax' + direction] as number,
```

Comment:
Please do not force cast to primitives. This is already of type `any`, so no additional cast is necessary.